docs/constants/navigation.js

```diff
@@ -351,6 +351,7 @@ export const general = [
         makePage('guides/building-for-tv.mdx'),
         makePage('guides/typescript.mdx'),
         makePage('guides/in-app-purchases.mdx'),
+        makePage('guides/notifications-services.mdx'),
```

Comment:
```suggestion
        makePage('guides/using-push-notifications-services.mdx'),
```

Can we update the page name so it reflects the updated title?